my-clerk-app/src/types/components.ts

```
@@ -0,0 +1,188 @@
+}
+
+export interface HeaderProps {
+  showMenu?: boolean;
+  onMenuClick?: () => void;
+}
+
+// Main App Components
+// eslint-disable-next-line @typescript-eslint/no-empty-object-type
+export interface HomePageProps {
+  // No props for main page component
+}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)